### PR TITLE
PIMS-281 Fix Agency Interest

### PR DESCRIPTION
**Pull request by Fosol**
Resolved issue with the agency response not working correctly.

image

### CODE REVIEW

File: frontend/src/components/tabs/styled/Tabs.tsx [hunk 1]
```diff
@@ -3,7 +3,7 @@ import styled from 'styled-components';
 export const Tabs = styled.div`
   padding-top: 1em;
 
-  ul {
+  & > ul {
```
Comment (Fosol) [inline frontend/src/components/tabs/styled/Tabs.tsx:6]: Fix styling issue with agency selection dropdown

File: frontend/src/features/projects/disposals/erp/AgencyInterest.tsx [hunk 2]
```diff
@@ -36,7 +36,7 @@ export const AgencyInterest = ({ disabled = false }: IAgencyInterestProps) => {
         project: values,
         agency: agency,
         note: '',
-        response: NotificationResponse.Watch,
+        response: AgencyResponses.Watch,
```
Comment (Fosol) [inline frontend/src/features/projects/disposals/erp/AgencyInterest.tsx:39]: Corrected type

File: frontend/src/index.scss [hunk 3]
```diff
@@ -30,3 +30,7 @@ input::-webkit-inner-spin-button {
     width: 100%;
   }
 }
+
+.form-control-lg {
+  font-size: 1em;
```
Comment (Fosol) [inline frontend/src/index.scss:35]: Fix styling issue from typeahead

**Comment by Fosol** [comment]
@Sharala-Perumal do you want to review this so that it can be merged into the dev branch?

**Review by Sharala-Perumal — APPROVED** [review]

**Comment by Sharala-Perumal** [comment]
> @Sharala-Perumal do you want to review this so that it can be merged into the dev branch?

@Fosol I have approved the changes now.

**Pull request merged**